<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QDialog" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="area1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#area1
{
	background-color:#242423;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton
{
        
        background-color: transparent;
		border-radius: 20px;
        color: #fff;
		text-align: left;
		padding-left: 50px;
}

#pushButton:hover
{
        background-color:#363634;
}</string>
   </property>
   <property name="text">
    <string>Inbox</string>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Logo {
background-color: transparent;
border-image: url(:Logo.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="area2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>-30</y>
     <width>901</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#area2
{
	background-color:#1c1c1b;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2
{
        
        background-color: transparent;
		border-radius: 20px;
        color: #fff;
		text-align: left;
		padding-left: 50px;
}

#pushButton_2:hover
{
background-color:#363634;
}</string>
   </property>
   <property name="text">
    <string>Starred</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_3
{
        
        background-color: transparent;
		border-radius: 20px;
        color: #fff;
		text-align: left;
		padding-left: 50px;
}

#pushButton_3:hover
{
background-color:#363634;
}</string>
   </property>
   <property name="text">
    <string>Sent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>336</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_4
{
        
        background-color: transparent;
		border-radius: 20px;
        color: #fff;
		text-align: left;
		padding-left: 50px;
}

#pushButton_4:hover
{
background-color:#363634;
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="inbox">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>150</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#inbox {
background-color: transparent;
border-image: url(:inbox.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="star">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>215</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#star {
background-color: transparent;
border-image: url(:star.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sent">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>279</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#sent {
background-color: transparent;
border-image: url(:sent.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="trash">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>336</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#trash {
background-color: transparent;
border-image: url(:trash.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-13</x>
     <y>128</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>410</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>20</width>
     <height>641</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_5
{
        
        background-color: transparent;
		border-radius: 20px;
        color: #fff;
		text-align: left;
		padding-left: 50px;
}

#pushButton_5:hover
{
background-color:#363634;
}</string>
   </property>
   <property name="text">
    <string>Work</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>485</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_6
{
        
        background-color: transparent;
		border-radius: 20px;
        color: #fff;
		text-align: left;
		padding-left: 50px;
}

#pushButton_6:hover
{
background-color:#363634;
}</string>
   </property>
   <property name="text">
    <string>Personal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_7
{
        
        background-color: transparent;
		border-radius: 20px;
        color: #fff;
		text-align: left;
		padding-left: 50px;
}

#pushButton_7:hover
{
background-color:#363634;
}</string>
   </property>
   <property name="text">
    <string>Credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="work">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>436</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#work {
background-color: transparent;
border-image: url(:folder_blue.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="personal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#personal {
background-color: transparent;
border-image: url(:folder_yellow.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="credit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>543</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#credit {
background-color: transparent;
border-image: url(:folder_green.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#search
{
    color:black;
	border-radius:20px;
	padding-left:30px;
}

#search:focus
{
border: 2px solid yellow;
}</string>
   </property>
   <property name="placeholderText">
    <string>Search...</string>
   </property>
  </widget>
  <widget class="QLabel" name="find">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>52</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#find {
background-color: transparent;
border-image: url(:search.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="sw">
   <property name="geometry">
    <rect>
     <x>1012</x>
     <y>42</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#sw
{
background-color: transparent;
background: none;
border: none;
background-repeat: none;
}

QCheckBox::indicator:unchecked {
    image: url(:moon.png);
}

QCheckBox::indicator:checked {
    image: url(:sun.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#user
{
	color:#fff;
}</string>
   </property>
   <property name="text">
    <string>Daniel Minulescu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>96</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_8
{
	    background-color: red;
		border-radius: 10px;
        color: #fff;
}


#pushButton_8:hover
{
	    background-color: #f72d2d;
}</string>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compose">
   <property name="geometry">
    <rect>
     <x>777</x>
     <y>36</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#compose {
background-color: transparent;
border-image: url(:compose.png);
background: none;
border: none;
background-repeat: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
	color:#fff;
}</string>
   </property>
   <property name="text">
    <string>Signed as:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
